employee-rostering-frontend/package.json

```diff
@@ -37,13 +37,21 @@
   "scripts": {
     "start": "mvn generate-sources; react-scripts start",
     "build": "mvn generate-sources; react-scripts build",
-    "typecheck": "npx tsc --noEmit",
-    "test": "TZ=UTC react-scripts test --transformIgnorePatterns \"node_modules/(?!(lodash-es))/\"",
+    "typecheck": "tsc --noEmit",
```

Comment:
I am assuming this is a deliberate error to verify it fails the build?

Reply:
I know we've discussed this on the chat but just for the record: This will actually fix the build on Jenkins where `npx` seems to be missing. Without `npx` in the script, when running `npm run typecheck`, `npm` will look for `tsc` in `node_modules/.bin` where it is provided by the `typescript` dependency.